Examples/UIExplorer/js/TextInputExample.android.js

```diff
@@ -623,4 +710,22 @@ exports.examples = [
     title: 'Toggle Default Padding',
     render: function(): ReactElement { return <ToggleDefaultPaddingExample />; },
   },
+  {
+    title: 'Text selection & cursor placement',
+    render: function() {
+      return (
+        <View>
+          <SelectionExample
+            style={styles.default}
+            value="text selection can be changed"
+          />
+          <SelectionExample
+            multiline
```

Comment:
Oh it's just a prop that is used to tell if we want to render that example with a normal or multiline text input. So what it does is renders the same example but one with single line and the other with multiline input.